polling/frequent/activities.py

```diff
@@ -17,6 +20,16 @@ async def compose_greeting(input: ComposeGreetingInput) -> str:
     while True:
         try:
             result = test_service.get_service_result(input)
+            print(f"exiting activity ${result}")
```

Comment:
I wonder if we should either remove the print statements or use a logger

```suggestion
            activity.logger.info(f"Exiting activity ${result}")
```

Reply:
fixed!